docs/index.rst

```diff
@@ -271,6 +272,9 @@ the stingray source directory.
 Using Stingray
 ===============
 
+The documentation below is built on top of Jupyter notebooks that can be run locally.
+The easiest way to retrieve the notebooks is by `cloning the notebooks repository <https://github.com/StingraySoftware/notebooks>`_ and browsing the directories, which are conveniently divided by topic.
+
```

Comment:
@taldcroft Is this addition in the right direction for what you were asking?

Reply:
Sounds good, thanks.